src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -430,7 +430,6 @@ function MdAutocomplete ($$mdSvgRegistry) {
                 aria-autocomplete="list"\
                 role="combobox"\
                 aria-haspopup="true"\
-                aria-activedescendant=""\
```

Comment:
The lighthouse a11y audit was complaining about these invalid aria values. They don't seem to be required by the spec and also don't seem to be used anywhere.

Reply:
This comes back in https://github.com/angular/material/pull/11441 but when there is no selected item, the attribute is removed as required by the spec.